util/csvresponse.py

```diff
@@ -14,6 +14,13 @@
 PRESSURE_DPI = app.config.get('PRESSURE_DPI')
 INT_PRESSURE_NAME = app.config.get('INT_PRESSURE_NAME')
 
+# constants for filtering QC Parameters
+QC_EXECUTED = 'qc_executed'
+QC_RESULTS = 'qc_results'
+
+# defeult parameters to request
+DEFAULT_PARAMETERS = ['time', 'deployment', 'lat', 'lon']
```

Comment:
these are the same parameters on the netcdf side of things?  maybe put them someplace common, update both netcdf, and csv to share.

Reply:
Will check that out.